internal/testutil/ibc/mock.go

```diff
@@ -10,34 +10,31 @@ import (
 )
 
 // BaseIBCMock returns a mock IBC module without a Host
-func BaseIBCMock(t gocuke.TestingT, busMock *mockModules.MockBus) *mockModules.MockIBCModule {
+func BaseIBCMock(t gocuke.TestingT, bus modules.Bus) *mockModules.MockIBCModule {
 	ctrl := gomock.NewController(t)
 	ibcMock := mockModules.NewMockIBCModule(ctrl)
 
 	ibcMock.EXPECT().Start().Return(nil).AnyTimes()
-	ibcMock.EXPECT().SetBus(busMock).Return().AnyTimes()
-	ibcMock.EXPECT().GetBus().Return(busMock).AnyTimes()
+	ibcMock.EXPECT().SetBus(bus).Return().AnyTimes()
+	ibcMock.EXPECT().GetBus().Return(bus).AnyTimes()
 	ibcMock.EXPECT().GetModuleName().Return(modules.IBCModuleName).AnyTimes()
+	ibcMock.EXPECT().HandleMessage(gomock.Any()).Return(nil).AnyTimes()
 
 	return ibcMock
 }
 
-// IBCMockWithHost returns a mock IBC module with a Host
-func IBCMockWithHost(t gocuke.TestingT, _ modules.EventsChannel) *mockModules.MockIBCModule {
+// BaseIBCHostMock returns a mock IBC Host submodule
+func BaseIBCHostMock(t gocuke.TestingT, busMock *mockModules.MockBus) *mockModules.MockIBCHostModule {
 	ctrl := gomock.NewController(t)
```

Comment:
(see: https://github.com/pokt-network/pocket/pull/868/commits/36d4d9d06899d8b45ba8da5a1a0d0452342b66af#r1252166693)

```suggestion
	t.Helper()

	ctrl := gomock.NewController(t)
```